### ISSUE TITLE
{widget:mode} / Image URL no longer works

### ISSUE THREAD

**Issue by johnjore** [igorgladkov/rotini#108]
With 0.6.50, "Image url="http://something/something.png" no longer works?

**Comment by igorgladkov** [comment]
Yes, as has been announced in G+ community, it has been removed to streamline other widget functionality. I'm working with users to find alternative solutions on case by case, so please describe your use case.

**Comment by johnjore** [comment]
I've only just started using rotini/openhab, but it looked to me as if it is manually set. I was hoping it would be extended to be possible to use rules to set the state automatically to the three states I use, Home, Away and Sleep, and keep the profile images.

**Comment by igorgladkov** [comment]
Is this something like a person "state"? If so, there will be a dedicated "person" widget released soon.

**Comment by johnjore** [comment]
Ah, that should fix it.
